ladybug_geometry/geometry2d/_2d.py

```diff
@@ -0,0 +1,84 @@
+# coding=utf-8
+"""Base class for 2D geometries in 2D space (Polygon2D and Mesh2D)."""
+from __future__ import division
+
+from .pointvector import Point2D
+
+
+class Base2DIn2D(object):
+    """Base class for 2D geometries in 2D space (Polygon2D and Mesh2D).
+
+    Properties:
+        vertices
+        min
+        max
+        center
+    """
+
+    @property
+    def min(self):
+        """A Point2D for the minimum bounding rectangle vertex around this geometry."""
+        if self._min is None:
```

Comment:
Yes, `self._min` gets set to `None` when you initialize the class, which happens when you initialize the Polygon2D or the Mesh2D that inherit from this class.  Because there's no `__init__` on this base class, there's no way to call any of these properties except through the classes that inherit from it.  Are you saying that we need to include an `__init__` here that we never actually use?  OR do we have to have duplicated versions of this property on each of the objects that inherit from it?

Reply:
> Are you saying that we need to include an __init__ here that we never actually use? OR do we have to have duplicated versions of this property on each of the objects that inherit from it?

Not sure what does that mean. In most cases you call the `__init__` from the baseclass when you initiate the subclass. Why will they be duplicated? 

```python
class A(object):
    def __init__(self, min=None, max=None):
        self._min = min
        self._max = max

class B(A):
    def __init__(self, min, max, length):
        A.__init__(self, min, max)
       self._length = length
```

Reply:
> In most cases you call the __init__ from the baseclass when you initiate the subclass.

I have not had a case where I have done this yet as I usually end up overwriting the `__init__` of the base class in the class that inherits from it since the inheriting class usually has additional properties that are not in the base class, which should usually be established in `__init__`

In any case, I can add an `__init__` to this base class in the event that someone tries to make a new class inheriting from it but doesn't overwrite `__init__`.  Because the base class here is hidden, I expect it to rarely happen but I guess it's better to be safe than sorry.